src/cloud-api/src/client.rs

```diff
@@ -70,13 +71,23 @@ impl Client {
         method: Method,
         path: P,
         cloud_provider: &CloudProvider,
+        api_version: Option<u16>,
```

Comment:
Can you document the behavior of what happens if we don't specify an `api_version`?